src/clients/python/__init__.py

```diff
@@ -161,6 +161,13 @@ def from_param(cls, value):
 _crequest_infer_ctx_input_set_raw = _crequest.InferContextInputSetRaw
 _crequest_infer_ctx_input_set_raw.restype = c_void_p
 _crequest_infer_ctx_input_set_raw.argtypes = [c_void_p, c_void_p, c_uint64]
+_crequest_infer_ctx_input_set_shared_memory = _crequest.InferContextInputSetSharedMemory
```

Comment:
We're not doing any python changes yet. Can you move this and crequest changes out of this commit.

Reply:
Sure. Will do